### DISCUSSION TITLE
Ability to "Freeze" parameters in a GP model?

### DISCUSSION THREAD

**Discussion by mathDR** [thomaspinder/GPJax#481] [General]
Hello!  Is there a way to "freeze" the ability of the optimizer to optimize parameters of the GP?  In particular, you could imagine we fit the GP to data then want to use that model as part of a larger model in `jax`.  But when we optimize the larger model, we want to keep the parameters of the GP "fixed".

Is there a performant way to do this in GPJax?  

In for example, in `BayesNewton` the constructor for a kernel [takes inputs](https://github.com/AaltoML/BayesNewton/blob/f72ae9a6ca69f9cce8e62135c9f78dda4825b4df/bayesnewton/kernels.py#L77-L78) called `fix_variance, fix_lengthscale`, etc.  that tells the model if those parameters are to be included in gradient calculations.

Thanks!

**Comment by thomaspinder** [comment]
Hi! 

Yes! You can do the below

```
import gpjax as gpx
from gpjax.parameters import Static

sigma_true = 0.2
likelihood = gpx.likelihoods.Gaussian(num_datapoints=n, obs_stddev=Static(sigma_true))
```
